### DISCUSSION TITLE
Ask user's browser to make an HTTP request and send the response to the Flet server.

### DISCUSSION THREAD

**Discussion by Hazelmeow** [flet-dev/flet#1356] [Q&A]
Sorry if this issue is duplicated.

**Comment by FeodorFitsner** [comment]
Could you please elaborate?

**Reply by Hazelmeow** [reply]
For example, when we call page.getResponseByUser("http://..."), then the Flet server will ask user's browser to make an HTTP request. After the browser get the response, it will send the response back to the Flet server. This feature might be good for separating frontend and backend.
